src/NuGet.Clients/NuGet.PackageManagement.UI/Models/DetailControlModel.cs

```diff
@@ -294,11 +378,48 @@ public DetailedPackageMetadata PackageMetadata
                 if (_packageMetadata != value)
                 {
                     _packageMetadata = value;
+
+                    if (_packageMetadata?.DeprecationMetadata != null)
+                    {
+                        PackageDeprecationReasons = ExplainPackageDeprecationReasons(_packageMetadata.DeprecationMetadata.Reasons?.ToList());
+                        PackageDeprecationCustomMessage = string.IsNullOrWhiteSpace(_packageMetadata.DeprecationMetadata.Message) ? null : _packageMetadata.DeprecationMetadata.Message;
+
+                        var alternatePackage = _packageMetadata.DeprecationMetadata.AlternatePackage;
+                        if (alternatePackage != null)
+                        {
+                            PackageDeprecationAlternatePackageText = GetPackageDeprecationAlternatePackageText(alternatePackage);
+                        }
+                    }
+
                     OnPropertyChanged(nameof(PackageMetadata));
+                    OnPropertyChanged(nameof(IsPackageDeprecated));
                 }
             }
         }
 
+        private string GetPackageDeprecationAlternatePackageText(AlternatePackageMetadata alternatePackageMetadata)
+        {
+            if (alternatePackageMetadata == null)
+            {
+                return null;
+            }
+
+            string text = alternatePackageMetadata.PackageId;
+
+            if (!alternatePackageMetadata.Range.IsFloating
+                && alternatePackageMetadata.Range.HasLowerBound)
+            {
+                text = $"{alternatePackageMetadata.PackageId} version {alternatePackageMetadata.Range.MinVersion.ToNormalizedString()}";
+            }
+
+            if (alternatePackageMetadata.Range.HasUpperBound)
+            {
+                text += Resources.Label_OrAbove;
```

Comment:
I'm curious how the alternative package view looks like in this scenario.

Reply:
This uncovered a [bug in db2catalog](https://github.com/NuGet/NuGet.Services.Metadata/pull/572). Hotfix pending deployment. Will share screenshot once fix is deployed.

Reply:
Verified hotfix on INT, all good.

The screenshot in the case of a package deprecation reason "Other" and alternate package version range defined as `[1.0.0, )`:

![image](https://user-images.githubusercontent.com/880728/60252354-606bbd00-98ca-11e9-8952-37671a6a14ae.png)

Reply:
Should we do similar to what we are doing in some places in the assets file? 

we have a type of string from VersionRange that is something like: 

`BaseTestPackage >= 1.0.0`

Thoughts @anangaur

Reply:
I tend to agree with this. @anangaur ?

Reply:
Ping @anangaur